plugins/modules/django_command.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: django_command
+author:
+  - Alexei Znamensky (@russoz)
+short_description: Run Django admin commands
+description:
+  - This module allows the execution of arbitrary Django admin commands.
+extends_documentation_fragment:
+  - community.general.django
+options:
+  command:
+    description:
+      - Django admin command. It must be a valid command accepted by C(python -m django) at the target system.
+    type: str
+    required: true
+  extra_args:
+    type: list
+    elements: str
+    description:
+      - List of extra arguments passed to the django admin command.
+"""
+
+EXAMPLES = """
+- name: Check the project
+  community.general.django_command:
+    command: check
+    settings: myproject.settings
+
+- name: Check the project in specified python path, using virtual environment
+  community.general.django_command:
+    command: check
+    settings: fancysite.settings
+    pythonpath: /home/joedoe/project/fancysite
+    venv: /home/joedoe/project/fancysite/venv
+"""
+
+RETURN = """
+run_info:
+  description: Command-line execution information (when verbosity >= 3)
+  type: dict
+  returned: success
```

Comment:
```suggestion
  description: Command-line execution information.
  type: dict
  returned: success and O(verbosity) >= 3
```

Reply:
I though the sanity check would allow only a fixed list of values in `returned` :o

Reply:
No, it only requires *a* text. Both fortunately (since it allows for more flexibility, like here, and unfortunately, since it allows random new creations for existing terms as well ;) ).